### PR TITLE
Feature camera

### PR DESCRIPTION
**Pull request by Shirel01**

### CODE REVIEW

File: camera.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+# I have to change with the consts
+
+
+
+def capture_5_images():
+    """
+    :return: list of frames
+    """
+
```
Comment (hodaya12) [inline camera.py:13]: unnecessary space

File: camera.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+            time.sleep(1)
+            if ret:
+                while (True):
+                    frame_list.append(frame)
+                    cv2.imshow("Frame", frame)  # show captured frame
+
+                    # press 'q' to break out of the loop
+                    if cv2.waitKey(1) & 0xFF == ord('q'):
+                        break
+                    if len(frame_list) == 5:
```
Comment (hodaya12) [inline camera.py]: change it with using const

File: camera.py [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    """
+    :return: list of frames
+    """
+
+    frame_list = []
+    # Create an object to hold reference to camera video capturing
+    vidcap = cv2.VideoCapture(0)
+
+    while True:
+
```
Comment (hodaya12) [inline camera.py:19]: unnecessary space

File: camera.py [hunk 4]
```diff
@@ -0,0 +1,53 @@
+import cv2
+import os
+import time
+
```
Comment (hodaya12) [inline camera.py]: import consts

**Comment by Shirel01** [comment]
i fixed things and i want to do new pull request

**Pull request closed without merging**